docs/glossary.md

```diff
@@ -41,54 +42,87 @@ Government Standard Image Build.
 
 <a id="internet-device">
 
-**Internet device**
+**Internet Device**
 
 </a>
 
 
-Device which is not a [GSIB](#gsib). This could be your personal device, or a device issued by your vendor or agency.
+Internet Device is not a [GSIB](#gsib). This could be your personal device, or a device issued by your vendor or agency.
+
+To access resources such as SGTS services and GCC via this device, you must onboard it to SEED (GovTech's MDM solution) .
 
-You must enroll this device with SEED (GovTech's MDM solution) in order to access SGTS resources, products and services that requires SEED.
+<a id="non-se-gsib">
 
 **non-SE GSIB**
 
+</a>
+
 If you are using a non-SE GSIB device, every time you log in to your device, you will be prompted to enter your BitLocker PIN. You can access the internet and intranet using this device.
 
-**Note**<br>
-> You can't onboard this device to SEED.
+You can't onboard a GSIB device to SEED.
+
+<a id="se-gsib">
 
 **SE GSIB**
 
+</a>
+
 It is a Secured Email (SE) GSIB device. This GSIB device is assigned to public officers who handle sensitive and secret information.
 
 If you are using a SE GSIB device, you will be using your PS-Card to authenticate. You will not be able to access internet using this SE-GSIB device.
 
 **Note**<br>
->- You can't use this device to activate your TechPass account.
+>- You can't use this device to sign up and onboard your TechPass account.
 >- You can't onboard this device to SEED.
 
+<a id="seed">
+
 **SEED**
 
+</a>
+
 Security Suite for Engineering Endpoint Devices. This is the MDM solution offered by GovTech.
 
+<a id="techpass">
+
 **TechPass**
 
+</a>
+
 This is an Identity & Access Management (IAM) and  a single sign-on(SSO) solution to seamlessly access Singapore Government Technology Stack(SGTS) services.
```

Comment:
1. We need to mention GCC?
2. FYI, in the near future we will TechPass accounts for Slack, Figma, etc. and non-citizen facing WoG apps.

Reply:
i will rephrase it in such a manner it covers both these cases.

This is an Identity & Access Management (IAM) and a single sign-on(SSO) solution to seamlessly access services such as Singapore Government Technology Stack(SGTS) and GCC.